Added GraphQL Playground starter

BlasiusSecundus opened the pull request:
Added GraphQL Playground starter. Currently only basic settings are available, but it is ready for testing and preliminary review.

oliemansm commented:
@BlasiusSecundus Thanks for the contribution! Looks pretty good to me. Anything else you want to do before we merge it?

BlasiusSecundus commented:
I have added two more changes. One small thing to normalize slashes, and a more complete test suite. With that I consider it ready for merging/initial release, so I removed the WIP prefix from the title.
